glide-core/README.md

```diff
@@ -1,5 +1,70 @@
 # Rust core
 
+## Prerequisites
+
+Please consider installing the following packages:
+
+- GCC
```

Comment:
I guess they didn't add all these to a common file because the idea was that people would prefer to look at the documentation of the specific client. (looking from a different perspective) Maybe, we should ask them if they prefer this before making the PR?